File: demo1.py
from sandstone import sandstone

class demo1(sandstone.application):
    def __init__(self, area, room):
        self.roomFile = "demo1/demo1Rooms.json"
        self.objectFile = "demo1/demo1Objects.json"
        super().__init__(area, room, self.roomFile, self.objectFile, saveFile="save.json")

demo1(1, 1).run()

File: demo1/src/demo1.cpp
#include <sandstone.h>

class Demo1 : public Sandstone::GameApplication
{
public:
	Demo1()
		: Sandstone::GameApplication("demo1Rooms.json", "demo1Objects.json")
	{
		
	}
	~Demo1()
	{

	}
};

Sandstone::Application* Sandstone::CreateApplication()
{
	return new Demo1();
}
